src/quote/swap/interfaces.ts

```diff
@@ -20,6 +20,7 @@ export interface SwapQuoteRequest {
   inputAmount?: string
   outputAmount?: string
   // Optional
+  address?: string
```

Comment:
If we'd try to use LiFi with new contracts at some point, we'd need to add address (user's account) to all requests.

Reply:
any reason why that would be a problem? (i'm wondering if we should just make it required starting now)

Reply:
because it changes several interfaces throughout the project which we might not need for any other swap quote provider